src/darkmode.js

```diff
@@ -1,17 +1,15 @@
 export default class Darkmode {
   constructor(options) {
```

Comment:
```suggestion
   constructor(options = {}) {
```

Then you can leave out all the `options &&` checks